spotbugsTestCases/build.gradle

```diff
@@ -41,3 +41,7 @@ task classesJava7(type:JavaCompile) {
 }
 
 tasks['build'].dependsOn classesJava7
+
+sonarqube {
+  skipProject true
```

Comment:
let's comment in this file, to tell your intent to developers even in the future